src/gpl/src/graphics.cpp

```diff
@@ -191,8 +191,38 @@ void Graphics::drawForce(gui::Painter& painter)
   }
 }
 
+void Graphics::drawCells(const std::set<GCell*, GCellComparator>& cells,
```

Comment:
Rather than copy the whole function just split out a drawCell(GCell*) and call it from both overloads of drawCells.